src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -264,7 +345,7 @@ class connectionStrings extends viewModelBase {
 
         if (sqlConnectionString) {
             if (this.isValid(sqlConnectionString.testConnectionValidationGroup)) {
-                eventsCollector.default.reportEvent("ravenDB-SQL-connection-string", "test-connection");
+                eventsCollector.default.reportEvent("SQL-connection-string", "test-connection");
```

Comment:
please avoid changing that - we already have events in GA - we want to correlate them!